### ISSUE TITLE
Can anyone tell my where i went wrong on this? I have been stuck on it all day

### ISSUE THREAD

**Issue by oliveruf42** [smartcontractkit/full-blockchain-solidity-course-py#957]
from solcx import compile_standard, install_solc

with open("./SimpleStorage.sol", "r") as file:
    simple_storage_file = file.read()

    # Compile Our Solidity
    install_solc("0.6.0")
    compiled_sol = compile_standard(
        {
            "language": "solidity",
            "source": {"SimpleStorage.sol": {"content": simple_storage_file}},
            "settings": {
                "ouputSelection": {
                    "*": {"*": ["abi", "metadata", "evm.bytecode", "evm.sourceMap"]}
                }
            },
        },
        solc_version="0.6.0",
    )
print(compiled_sol)

  Eddies-iMac:web3_py_simple_storage eddiebraddock$ Python3 deploy.py
Traceback (most recent call last):
  File "/Users/eddiebraddock/demos/web3_py_simple_storage/deploy.py", line 8, in <module>
    compiled_sol = compile_standard(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/solcx/main.py", line 365, in compile_standard
    raise ContractsNotFound(
solcx.exceptions.ContractsNotFound: Input JSON does not contain any sources
> command: ``
> return code: `None`
> stdout:
None
> stderr:
None
Eddies-iMac:web3_py_simple_storage eddiebraddock$

**Comment by oliveruf42** [comment]
When I used deploy.py earlier in the course it worked fine. Now I can't get this to work and i have been stuck for almost 24 hours. I have tried every troubleshoot on Stackoverflow. I've tried using Python deploy.py, Python3 deploy.py. I've set up and used a virtual env to try the same. I've double checked that i have all the correct packages installed.

Any help would be greatly appreciated. Thanks in advance
